webapp/src/marked.tsx

```diff
@@ -342,9 +342,7 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
         markdown = markdown.replace(/<\s*script[^>]*>.*<\/\s*script\s*>/g, '');
 
         // Render the markdown and add it to the content div
-        /* tslint:disable:no-inner-html (marked content is already sanitized) */
         content.innerHTML = marked(markdown);
```

Comment:
This rule?
 // eslint-disable-next-line @microsoft/sdl/no-inner-html

Reply:
For some reason that rule doesn't complain here. Maybe this is actually safe? Or the rule got less good at checking :/

![2021-04-02 10 18 07](https://user-images.githubusercontent.com/6453828/113438164-ddc83980-939c-11eb-8f50-a4a72b5f94e7.gif)

Reply:
Ooooh, it looks like innerHTML is actually safe on `<div>`, `<span>`, and `<noembed>`.
TIL: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML